cirq/sim/clifford/clifford_simulator.py

```diff
@@ -346,6 +344,26 @@ def apply_unitary(self, op: 'cirq.Operation'):
                              str(op.gate))  # type: ignore
         return
 
+    def apply_measurement(self,
+                          op: 'cirq.Operation',
+                          measurements: Dict[str, List[np.ndarray]],
+                          prng: np.random.RandomState,
+                          collapse_state_vector=True):
+        if collapse_state_vector:
+            state = self
+        else:
+            state = self.copy()
+
+        key = cirq.measurement_key(op)
+        qids = [self.qubit_map[i] for i in op.qubits]
+
+        args = clifford.ActOnCliffordTableauArgs(state.tableau, qids, prng,
+                                                 measurements)
+        act_on(op, args)
+
+        for i, qid in enumerate(qids):
+            state.ch_form.project_Z(qid, measurements[key][i])
+
     @deprecated_parameter(
```

Comment:
Given that the whole function is deprecated now, should I remove the argument deprecation notice?